lib/media/track/localaudiotrack.js

```diff
@@ -183,7 +189,7 @@ class LocalAudioTrack extends LocalMediaAudioTrack {
   }
 
   /**
-   * Disable the {@link LocalAudioTrack}. This is effectively "mute".
```

Comment:
I think if we remove `This is effectively "mute"`, perhaps we need to be more clear about what the differences are between mute/unmute and disable/enable are.

Perhaps adding 
```This is effectively "pause"```